### DISCUSSION TITLE
Mistake, Chapter 7 section 2 exercise 7b.

### DISCUSSION THREAD

**Discussion by levdopa** [spamegg1/Epp-Discrete-Math-5th-solutions#13] [General]
G(a) = G(b), so G is not one-to-one, but you write they don't equal each other and G is one-to-one.

**Comment by spamegg1** [comment]
Thanks! Fixing it now.

**Comment by spamegg1** [comment]
Fixed [here](https://github.com/spamegg1/Epp-Discrete-Math-5th-solutions/commit/14f194f5570ba00eda27ccf21d85234a04bc89b9)
